jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -89,6 +89,12 @@ static void checkDeprecatedUsage(JibExtension jibExtension, Logger logger) {
             "You cannot configure both 'jib.extraDirectory.path' and 'jib.extraDirectories.paths'");
       }
     }
+
+    if (jibExtension.getContainer().getUseCurrentTimestamp()) {
+      logger.warn(
+          "'jib.container.useCurrentTimestamp' is deprecated; use 'jib.container.creationTime' to "
```

Comment:
People will use `useCurrentTimestamp` to set the current time. The message should also include a way to achieve this, like using some constant like `USE_CURRENT_TIMESTAMP`.

Reply:
Should the same constant be available for `filesModificationTime`?

Reply:
That's viable. I'll file a new issue.

Reply:
Ok, I can still take care of it in this PR since they share this code path.

Reply:
I'd like to hold on on that. Created #1889 and explained the reason.

Reply:
I think the message should include the keyword so that people know what to do to preserve the behavior of `<useCurrentTimestamp>`

"... to specify an ISO 8601 date-time or a special keyword ..."

Reply:
Yeah, doing that now, forgot to do that on the gradle side.